src/engine/computer_modern_data.jl

```diff
@@ -226,17 +220,22 @@ _latex_to_bakoma = Dict(
     raw"\uplus"                    => ("cmsy10", 0x5d),
     raw"\vdash"                    => ("cmsy10", 0x60),
     raw"\vee"                      => ("cmsy10", 0x5f),
-    raw"\vert"                     => ("cmsy10", 0x6a),
     raw"\wedge"                    => ("cmsy10", 0x5e),
     raw"\wr"                       => ("cmsy10", 0x6f),
-    raw"\|"                        => ("cmsy10", 0x6b),
     raw"|"                         => ("cmsy10", 0x6a),
-    raw"\_"                        => ("cmtt10", 0x5f)
+    raw"_"                         => ("cmtt10", 0x5f)
 )
 
 
-const BakomaSymbolSet = SymbolSet(
-    "Bakoma (Computer Modern)",
-    Dict([sym => TeXChar(Char(code), join("assets/fonts/$font.ttf"))
-          for (sym, (font, code)) in _latex_to_bakoma])
-)
\ No newline at end of file
+_symbol_to_computer_modern = Dict{Char, Tuple{String, Char}}()
+
+for (symbol, (fontname, id)) in _latex_to_computer_modern
+    if haskey(latex_symbols, symbol)
+        symbol = only(latex_symbols[symbol])
+    else
+        symbol = only(symbol)
```

Comment:
@Kolaru `only` is only available in Julia >= 1.4. Was it intended to drop support for Julia 1.3? It seems that Project.toml does not reflect this and hence the compatibilities in the registry are not correct (on Julia 1.3 the package can be installed but not precompiled). I assume it went unnoticed since you don't run CI with Julia 1.3.

Is it necessary to use `only` here? If the check is necessary you could just add a `length(symbol) == 1 || error("...")` statement after the if block.

Reply:
Absolutely not, I just was sloppy (with CI too so tests were not run on v1.3).

 #41 Will fix that and I'll tag a new version as soon as the tests have passed.

Reply:
Thank you 👍